sdk/storage/Azure.Storage.Blobs.Batch/src/ContainerRestClient.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using Azure.Core;
+
+namespace Azure.Storage.Blobs.Batch
+{
+    internal partial class ContainerRestClient
+    {
+        // We are overriding this method because the new generator attempts to add 2 content types to the Content-Type header,
+        // causing auth to fail.  https://github.com/Azure/autorest/issues/3914
+        internal HttpMessage CreateSubmitBatchRequest(string containerName, long contentLength, string multipartContentType, Stream body, int? timeout)
```

Comment:
Could you please create issue in our repo that tracks that as well? We should be able to remove this customization when generator work is done.

Reply:
fixed.